draft-yasskin-dispatch-web-packaging.md

```diff
@@ -0,0 +1,650 @@
+---
+coding: utf-8
+
+title: Web Packaging
+docname: draft-yasskin-dispatch-web-packaging-latest
+category: std
+
+ipr: trust200902
+area: gen
+workgroup: Dispatch
+keyword: Internet-Draft
+
+stand_alone: yes
+pi: [comments, sortrefs, strict, symrefs, toc]
+
+author:
+ -
+    name: Jeffrey Yasskin
+    organization: Google
+    email: jyasskin@chromium.org
+
+normative:
+  CBOR: RFC7049
+  CDDL: I-D.greevenbosch-appsawg-cbor-cddl
+  appmanifest: W3C.WD-appmanifest-20170608
+  SRI: W3C.REC-SRI-20160623
+
+informative:
+  ServiceWorkers: W3C.WD-service-workers-1-20161011
+
+--- abstract
+
+Web Packages provide a way to bundle up groups of web resources to
+transmit them together. These bundles can then be signed to establish
+their authenticity.
+
+--- middle
+
+Introduction        {#problems}
+============
+
+People would like to use content offline and in other situations where
+there isn’t a direct connection to the server where the content
+originates. However, it's difficult to distribute and verify the
+authenticity of applications and content without a connection to the
+network. The W3C has addressed running applications offline with
+Service Workers ({{ServiceWorkers}}), but not
+the problem of distribution.
+
+We've started work on this problem
+in [](https://github.com/WICG/webpackage), but we suspect that the
+IETF may be the right place to standardize the overall format. More
+details can be found in that repository.
+
+Use Cases    {#use-cases}
+---------
+
+### Offline Installation
+
+People with expensive or intermittent internet connections are used
+to sharing files via P2P links and shared SD cards. They should be
+able to install web applications they received this way. Installing a
```

Comment:
I don't know a well-defined term for this. "Install" doesn't work well with "websites", which also isn't particularly defined. "Service workers" have a defined install step, but it's not obvious that we want to insist on service workers for these packages. Any suggestions?